src/network/networkctl.c

```diff
@@ -2585,7 +2588,7 @@ static int lldp_neighbors_varlink_reply(Varlink *link, JsonVariant *parameters,
                 return r;
 
         if (udata->table) {
-                capabilities = lldp_capabilities_to_string(entry.capabilities);
+                capabilities = lldp_capabilities_to_string(entry.capabilities, true);
```

Comment:
as per current coding style, if you have function parameters that give no indication what they are about please prefix them with a comment. i.e. change `true` to `/* dots= */ true` here

Reply:
Done, sorry.